docs/storage.md

```diff
@@ -61,8 +61,12 @@ A Prometheus server's data directory looks something like this:
 Note that a limitation of local storage is that it is not clustered or
 replicated. Thus, it is not arbitrarily scalable or durable in the face of
 drive or node outages and should be managed like any other single node
-database. The use of RAID is suggested for storage availability, and
-[snapshots](querying/api.md#snapshot) are recommended for backups. With proper
+database. 
+
+The use of RAID is suggested for storage availability, and
+[snapshots](querying/api.md#snapshot) are recommended for backups. Backups 
```

Comment:
I kinda feel like we should drop the RAID thing. Considering there are other durability/availability options like ZFS today.

```suggestion
[Snapshots](querying/api.md#snapshot) are recommended for backups. Backups 
```